### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by FarukhS52**
# Pull Request

Include a summary of the changes.

## Type of change

Delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How has this change been tested?

Describe how this code was/is tested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Not only that, Python has become the _de facto lingua franca_ of data science, m
 
 The language out of the box is not without its drawbacks however, especially in the context of
 implementing large performance-critical systems. Cython has addressed a lot of these issues, offering all the advantages
-of a statically typed language, embedded into Pythons rich ecosystem of software libraries and
+of a statically typed language, embedded into a Python-rich ecosystem of software libraries and
```
Comment (cjdsellers) [inline README.md:92]: The wording here changes the meaning.

File: docs/integrations/betfair.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ When starting the trading node, you'll receive immediate confirmation of whether
 credentials are valid and have trading permissions.
 
 :::tip
-Best practice is to manage your credentials using environment variables.
+The best practice is to manage your credentials using environment variables.
```
Comment (cjdsellers) [inline docs/integrations/betfair.md:58]: Probably more concise to keep the original.

File: docs/integrations/bybit.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ and won't need to necessarily work with these lower level components directly.
 
 ## Bybit documentation
 
-Bybit provides extensive documentation for users which can be found in the [Bybit help center](https://www.bybit.com/en/help-center).
+Bybit provides extensive documentation for users which can be found in the [Bybit help centre](https://www.bybit.com/en/help-center).
```
Comment (cjdsellers) [inline docs/integrations/bybit.md:47]: Lets stick with American English for consistency ("center").

File: docs/integrations/polymarket.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ A single wallet address is supported per trader instance when using environment
 or multiple wallets could be configured with multiple `PolymarketExecutionClient` instances.
 
 :::info
-Ensure your wallet is funded with **USDC.e**, otherwise you will encounter the "not enough balance / allowance" API error when submitting orders.
+Ensure your wallet is funded with **USDC.e**, otherwise you will encounter the "not enough balance/allowance" API error when submitting orders.
```
Comment (cjdsellers) [inline docs/integrations/polymarket.md:82]: The original is the exact error message from Polymarket, so best to stick with that I think.

**Review by cjdsellers — COMMENTED** [review]
Good updates, just a couple of minor comments below.

**Comment by FarukhS52** [comment]
@cjdsellers Thanks for the feedback. I have made the changes, thank you .

**Review by cjdsellers — APPROVED** [review]

**Pull request merged**